### DISCUSSION TITLE
Hobby Account Inquiry

### DISCUSSION THREAD

**Discussion by Awesomedonut** [vercel/community#5595] [Account Issue] [answered]
### Question

I'd like to use the features of the Hobby Account (ie deploy a github repo for free) but I don't see the "Hobby" label on my account. I asked the question [here](https://github.com/orgs/vercel/discussions/5485#discussioncomment-8187295) but the discussion was closed without my issue being resolved.

<img width="431" alt="Screenshot 2024-01-24 at 3 45 51 PM" src="https://github.com/vercel/community/assets/12404874/7628a2b5-70e7-4f3c-a428-a508b4b9d905">

I was told to just create a team, but as you can see below, there are no free team options. Are there any other solutions so that I can deploy a github repo? Thanks!

<img width="570" alt="Screenshot 2024-01-24 at 3 45 55 PM" src="https://github.com/vercel/community/assets/12404874/29fd8074-88c9-4db4-a6c9-46d1c00f230d">

**Comment by amyegan** [comment, marked as answer]
Hi @Awesomedonut. There is no charge for a Pro Trial, so you can create a new team with that and downgrade it to Hobby at the end of the two week trial. Please let me know if you run into any trouble with this workaround 🙏
